City-of-Helsinki/drupal-hdbt-admin
# UHF-218 Admin theme adjustments

To test:

1. `composer create-project City-of-Helsinki/drupal-helfi-platform:dev-main yoursite --no-interaction --repository https://repository.drupal.hel.ninja/`
2. Switch to corresponding branch: `composer require drupal/hdbt_admin:dev-UHF-218-admin-theme-adjustments`
3. Run `make new`
4. Log in to the site.
5. Go to https://yoursite.docker.sh/fi/node/add/page and make sure the admin interface looks sleek and functional and that there is no bugs. Make sure it also works responsively.
6. Go to https://yoursite.docker.sh/fi/node/1/edit and make sure the admin interface looks sleek and functional and that there is no bugs. Make sure it also works responsively.
7. All-in-all test out the admin user interface and try to make sure that there is no more large visual bugs and that things are in logical places.
